libraries/ae-test-common/src/main/java/org/metaeffekt/core/itest/common/setup/UrlBasedTestSetup.java

```diff
@@ -101,8 +112,8 @@ public boolean load(boolean overwrite) throws IOException {
         String filename = filenameParts[filenameParts.length - 1];
         String artifactFile = getDownloadFolder() + filename;
         if (overwrite || !new File(artifactFile).exists()) {
-            new File(getDownloadFolder()).mkdirs();
             if (url.startsWith("http")) {
+                new File(getDownloadFolder()).mkdirs();
```

Comment:
Use FileUtils.mkdir signatures.

Reply:
you mean `FileUtils.forceMkdir(new File(getDownloadFolder()));` by this?

Reply:
Yes